### ISSUE TITLE
Binary Cross Entropy Loss with Logits or with Probabilities

### ISSUE THREAD

**Issue by danzhang0615** [wysoczanska/clip_dinoiser#22]
Hi,

https://github.com/wysoczanska/clip_dinoiser/blob/main/train.py/#L42 I notice here the loss is taking the logits as the inputs, but corr should be interpreted as probabilities, also in the paper. Can you please clarify, if the loss definition in the code is correct or not.

Thanks.
